src/models/config/ChatAnalyticsConfig.ts

```diff
@@ -6,8 +9,8 @@
 export interface ChatAnalyticsConfig {
   /** The API key for accessing the Analytics Events API. */
   apiKey: string; // TODO: Add optional "token" for JWT auth
-  /** The environment to send the requests to. */
-  env?: 'PRODUCTION' | 'SANDBOX';
-  /** The physical region of the Yext account */
-  region?: 'US' | 'EU';
+  /** {@inheritDoc SearchAnalyticsConfig.env} */
+  env?: Environment;
```

Comment:
can we do EnumOrLiteral for env and region?

Reply:
updated!